ebuild-maintenance/general/text.xml

```diff
@@ -0,0 +1,111 @@
+<?xml version="1.0"?>
+<guide self="ebuild-maintenance/general/">
+<chapter>
+<title>Maintaining Your Package</title>
+
+<section>
+<title>General responsibilities</title>
+<body>
+
+<note>
+Gentoo's Quality Assurance (QA) team also maintains a
+<uri link="https://projects.gentoo.org/qa/policy-guide/">policy guide</uri>,
+which contains useful advice on ebuild structure and behaviour.
+</note>
+  
+<p>
+You should always use the latest <uri link="::ebuild-writing/eapi/">EAPI</uri>.
+
+The main exception is for critical packages like Portage or
+those typically maintained by base-system which normally wait
+~2-3 years.
+
+Note that for new EAPIs, we require 1 year before using them in
+stable ebuilds.
```

Comment:
What does this mean? If EAPI-8 was pushed today, no EAPI-8 ebuild was going to be stabilized for a year? That doesn't seem right.